README.md

```diff
@@ -5,63 +5,69 @@
 
 - **HierarchicalXC**: A hierarchical clustering based approach
 
-## Using the module
+This project also includes scripts for training and testing on datasets using this module.
 
-### Setup
+## Setup
 
-- Requirements for the project are listed in `requirements.txt`. In addition to these, it is mandatory to have PyTorch installed in your system with version 0.4.1 or higher.
-   - Requirements can be installed using `pip`:
-     ```bash
-     $ pip install -r requirements.txt 
-     ```
-     or using `conda`:
-     ```bash
-     $ conda install --file requirements.txt
+### Prerequisites
+
+- Python 2.7 or 3.5
+- Requirements for the project are listed in [requirements.txt](requirements.txt). In addition to these, PyTorch 0.4.1 or higher is necessary. The requirements can be installed using pip:

```

Comment:
`[requirements.txt](https://github.com/vishwakftw/extreme-classification/blob/master/requirements.txt)` would be better I suppose. I am not sure if the hyperlinks work this way too, so please feel free to omit this suggestion.

Reply:
They work. The advantage of this way is that it will work irrespective of the branch you are in - it will redirect to the current branch's version.

Reply:
Ah that's good then. Thanks.